nixos/tests/nextcloud/basic.nix

```
@@ -1,4 +1,4 @@
-args@{ pkgs, nextcloudVersion ? 22, ... }:
+args@{ pkgs, nextcloudVersion ? 28, ... }:
```

Maybe we should drop this completely because it wasn't updated for ages.

```suggestion
args@{ pkgs, nextcloudVersion, ... }:
```